samples/WebAPIMigration/ProductsDnx/Startup.cs

```diff
@@ -25,6 +25,15 @@ public void Configure(IApplicationBuilder app, IHostingEnvironment env)
             app.UseStaticFiles();
             // Add MVC to the request pipeline.
             app.UseMvc();
+
```

Comment:
As mentioned previously, we recommend using attribute routing instead of trying to make things work with global routes.